client/src/pages/Forum.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { useQuery } from "@apollo/client";
 
 import PostList from "../components/PostList";
-import PostForm from "../components/postForm";
```

Comment:
I changed "PostForm" to "postForm" since github changes the case of the folder. so what was there before should be right.